site/content/en/docs/tasks/run_rayjobs.md

```diff
@@ -139,4 +184,9 @@ spec:
                     cpu: "1"
 ```
 
-The provided example is based on KubeRay's [RayJob sample](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray_v1alpha1_rayjob.yaml), the target queue `kueue.x-k8s.io/queue-name: user-queue` being specified in the labels and `shutdownAfterJobFinishes` set to`true`.
+You can run this RayJob with the following command:
+
+```sh
+kubectl apply -f rayjob-code-sample.yaml
+kubectl create -f sample-ray-job.yaml
```

Comment:
```suggestion
# create the code config map (once)
kubectl apply -f ray-job-code-sample.yaml

# create a ray job, this can be run multiple times to observe 
# the serial admission of the jobs.
kubectl create -f ray-job-sample.yaml
```

Reply:
Done